templates/ui.html

```diff
@@ -2131,9 +2160,45 @@
           }
         },
         mounted() {
-          this.updateDefaultOptions("default", this.initializeFilerobot);
+          this.updateDefaultOptions(
+              "default",
+              () => {this.initializeFilerobot, this.startIntroJS()}
+          );
         },
       }).use(vuetify).mount('#app');
     </script>
+    <script>
+      function getIntroJS() {
+        return introJs().setOptions({
+          tooltipClass: "text-black",
+          showProgress: true,
+          showBullets: false,
+          steps: [
+            {
+              title: "Fooocus Quick Guide",
+              element: document.querySelector("#prompt-textarea"),
+              intro: "Input your prompt here.",
```

Comment:
I would suggest to change it to "Just simply describe what you would like to see in plain English"